src/resources/i18n/translations/en.json

```
@@ -6,7 +6,7 @@
   "15 minutes": "15 minutes",
   "18 per year after that": "$18/year thereafter",
   "2 hours": "2 hours",
-  "3 months free": "3 months for free",
+  "3 months free": "3 months for as a gift to you",
```

Sorry, I'm going to change this back. It just seems natural to my ears to hear "for free" meaning "no charge", as in "you don't have to pay money". This is universally understood in the US, and we don't want confusion here about the membership's "free trial" terms.

---

I meant to not do "for" here.
```suggestion
  "3 months free": "3 months as a gift to you",
```
The psychology is that one is more likely to pay for a service when the idea is an implied or perceived cost, rather than something that is without cost (free). Also there is some sense of indebtedness, which may help.
"For free" could still be added, but I imagine that would be too long.